x/projects/README.md

```diff
@@ -0,0 +1,219 @@
+}
+```
+
+The project index is: `<subscription_owner_address>-<project_name>` (for example: `lava@1p7ezdua4ma90858h7zy9727wmqy53vk0mkdlvg-myProject`). When a consumer buys a subscription, an admin project is automatically generated with the index `<subscription_owner_address>-admin`.
+
+The project is subject to three types of policies: the plan's policy, the subscription's policy, and the admin policy. The plan's policy cannot be changed using transactions (only via a government proposal), while the other two policies can be changed using transactions. The effective policy that the project is subject to is the strictest policy calculated from all three policies. To query for the project's effective policy, use the `Pairing` module's query: `effective-policy` (see [below](#queries)).
+
+Projects are saved in a fixation store (see `FixationStore` module for reference). By using the snapshot ID, we can differentiate between different versions of the project.
+
+For more details regarding the `Policy` struct and its limitations, see [here](https://github.com/lavanet/lava/blob/main/x/plans/README.md#policy). For more details about `ProjectKeys`, see [below](#project-keys).
+
+### Project Keys
+
+A project has a list of accounts (project keys) that can use the project in different ways. There are two kinds of keys: admin key and developer key. A project key is defined as follows:
+
+```go
+type ProjectKey struct {
+	Key    string  // user lava address
+	Kinds  uint32  // key kind
+}
+```
+
+The `Kinds` enum is defined as follows:
+
+```
+enum Kinds {
+    ADMIN = 1            // admin key
+    DEVELOPER = 2        // developer key
+    ADMIN+DEVELOPER = 3  // admin+developer key
+}
+```
+
+The admin key lets the admin edit the project's admin policy, enable/disable the project and add/delete other project keys. A single user can be admin in multiple projects. The subscription owner is automatically considered an admin in all of the project created under it.
+
+The developer key lets a user to use the project, i.e., make RPC calls and use the project's CU. A single user (consumer) can only be a developer in one project. The project's developer can be viewed as a dapp developer that wants to use Lava to get RPC data for their dapp. To allow end users use their project to get RPC data, the dapp developer will need to use badges. For more details on badges, see [below](#badges).
+
+Note that the admin cannot use the project's CU like a developer, they can only edit the project's properties.
+
+### Badges
+
+Badges are a method to grant compute units (CU) from projects to other users. Given a subscription with projects, a developer key (in a project) can be used to generate a badge key, to be signed by an external badge-server. The badge is ephemeral, limited in time and in CU capacity; It can be used to get pairing with providers, and for requests from providers - which will be charged to the respective project after verification.
```

Comment:
Can you rephrase the last sentence here, it is not very clear.
(Starting from "It can be used to get...")

Reply:
hope it's clearer now. see https://github.com/lavanet/lava/commit/189b8756e705e6ccd58fd7e92b7231078bd5e5e0

Reply:
Perfect!